docs/usage/logging.md

```diff
@@ -17,25 +17,23 @@ The logging stack is extended to scrape logs from the systemd services of each s
 Also, in the shoot control plane an `event-logger` pod is deployed which scrapes events from the shoot `kube-system` namespace and shoot `control-plane` namespace in the seed. The `event-logger` logs the events to the standard output. Then the `fluent-bit` gets these events as container logs and sends them to the Loki in the shoot control plane (similar to how it works for any other control plane component).
 
 ### How to access the logs
-The first step is to authenticate in front of the Grafana ingress.
-There are two Grafana instances where the logs are accessible from.
-  1. The user (stakeholder/cluster-owner) Grafana consist of a predefined  `Monitoring and Logging` dashboards which help the end-user to get the most important metrics and logs out of the box. This Grafana UI is dedicated only for the end-user and does not show logs from components which could log a sensitive information. Also, the `Explore` tab is not available. Those logs are in the predefined dashboard named `Controlplane Logs Dashboard`.
```

Comment:
The `Controlplane Logs Dashboard` still exists and is broken with the changes from this PR.

Reply:
We double checked that the `Controlplane Logs Dashboard` works:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/23032437/197156590-d0c09e88-8140-4592-982b-aa6e9eebf022.png">

Note that Grafana/loki requires the `X-Scope-OrgID operator` header and that is set by the ingress, so port-forwarding to Grafana is not enough, we have to go through the ingress. If you are using the remote local setup, these are the steps:

```
kubectl port-forward -n garden svc/nginx-ingress-controller 1443:443

k get secrets -n garden-local local.monitoring -o json | jq .data.password -r | base64 -d; echo

# on the laptop
k port-forward deployments/remote-local-setup 1443
open https://gu-local--local.ingress.local.seed.local.gardener.cloud:1443/?orgId=1

# in /etc/hosts, have an entry like this
127.0.0.1 *.ingress.local.seed.local.gardener.cloud
```

Reply:
I think my issue was rather that the logging was not enabled for the Seed. Sorry about this comment that is not applicable.